share/completions/climate.fish

```diff
@@ -0,0 +1,82 @@
+# fish completion for climate
+
+function __fish_is_git_repository
+    git rev-parse --is-inside-work-tree ^/dev/null >/dev/null
+    return $status
+end
+
+function __fish_climate_already_has_argument
+    set cmd (commandline -opc)
+    if not set -q cmd[2]
```

Comment:
You don't need the `if` statement with the `return`s.

Just write: `not set -q cmd[2]`